+++
title = "iCount"
linkTitle = "iCount"

+++

{{< blocks/cover title="Welcome to iCount: a package for protein-RNA interaction analysis" image_anchor="top" height="full" color="orange" >}}
<div class="mx-auto">
	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
	</a>
	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/tomazc/iCount">
		Download <i class="fab fa-github ml-2 "></i>
	</a>
	<p class="lead mt-5"><b>iCLIP</b> data processing - in the cloud!</p>
	{{< blocks/link-down color="info" >}}
</div>
{{< /blocks/cover >}}


{{% blocks/lead color="primary" %}}

<div class="text-left">
iCount is a Python module and associated command-line interface (CLI),
which provides all the commands needed to process iCLIP data on
protein-RNA interactions and generate:
<br><br>

+ demultiplexed and adapter-trimmed FASTQ files,
+ BAM files with mapped iCLIP reads,
+ identified protein-RNA cross-linked sites, saved to BED files,
+ peaks composed of statistically significant cross-linked sites, saved to BED files,
+ clusters of significant cross-linked sites, saved to BED files,
+ grouping of individual replicate experiments,
+ RNAmap generation showing the positional distribution of cross-linked sites relative to genomic landmarks,
+ kmer enrichment analysis,
+ and other.

You may start with the [tutorial](http://icount.readthedocs.io/en/latest/tutorial.html) or dive into the
[documentation](http://icount.readthedocs.io/en/latest/index.html).

</div>

{{% /blocks/lead %}}

{{< blocks/section color="dark" >}}
{{% blocks/feature icon="fa-lightbulb" title="New chair metrics!" %}}
The Goldydocs UI now shows chair size metrics by default.

Please follow this space for updates!
{{% /blocks/feature %}}


{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/google/docsy-example" %}}
We do a [Pull Request](https://github.com/google/docsy-example/pulls) contributions workflow on **GitHub**. New users are always welcome!
{{% /blocks/feature %}}


{{% blocks/feature icon="fab fa-twitter" title="Follow us on Twitter!" url="https://twitter.com/docsydocs" %}}
For announcement of latest features etc.
{{% /blocks/feature %}}


{{< /blocks/section >}}


{{< blocks/section >}}
<div class="col">
<h1 class="text-center">This is the second Section</h1>
</div>

{{< /blocks/section >}}



{{< blocks/section >}}
{{% blocks/feature icon="fab fa-app-store-ios" title="Download **from AppStore**" %}}
Get the Goldydocs app!
{{% /blocks/feature %}}


{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/google/docsy-example" %}}
We do a [Pull Request](https://github.com/google/docsy-example/pulls) contributions workflow on **GitHub**. New users are always welcome!
{{% /blocks/feature %}}


{{% blocks/feature icon="fab fa-twitter" title="Follow us on Twitter!" url="https://twitter.com/GoHugoIO" %}}
For announcement of latest features etc.
{{% /blocks/feature %}}


{{< /blocks/section >}}

{{< blocks/section >}}

<div class="col-12">
<h1 class="text-center">This is another Section</h1>
</div>

{{< /blocks/section >}}
